CHANGES.rst

```diff
@@ -27,6 +27,7 @@ Fixed
 
 - Fix documentation build error due to deleted link definition to deprecated
   `debops.php5` role repository. [ganto_]
+- Probe if `roundcube__domain` is a string and construct `roundcube__git_checkout` accordingly.  [cultcom]
```

Comment:
As @ypid said, you should use the `envvar` reference for variables. See e.g. https://github.com/debops-contrib/ansible-roundcube/blame/master/CHANGES.rst#L52 for an example.